docs/main_text.md

```diff
@@ -433,6 +432,13 @@ We then quantile normalize all samples as described above.
 
 We've made our analyses underlying processing choices and exploring test compendia available at our <a href = "https://github.com/AlexsLemonade/compendium-processing" target = "blank">`compendium-processing`</a> repository.
 
+# API
+
+You can use the refine.bio API to build your own applications. 
```

Comment:
utilizing the refine.bio processed data. 
You can also customize your download with options available through our API like, obtaining experiment/sample specific quant files. 

How does that sound?

Reply:
Take a look at https://github.com/AlexsLemonade/refinebio-docs/pull/110/commits/0f0f7065a9a1b303682d0936ab37e3894de40617